Core/src/org/sleuthkit/autopsy/casemodule/CaseMetadata.java

```diff
@@ -452,13 +463,28 @@ private void readFromFile() throws CaseMetadataException {
             }
             Element caseElement = (Element) caseElements.item(0);
             this.caseName = getElementTextContent(caseElement, CASE_NAME_ELEMENT_NAME, true);
+            String caseDisplayName;
+            String caseNumber;
             if (schemaVersion.equals(SCHEMA_VERSION_ONE) || schemaVersion.equals(SCHEMA_VERSION_TWO)) {
-                this.caseDisplayName = caseName;
+                caseDisplayName = caseName;
             } else {
-                this.caseDisplayName = getElementTextContent(caseElement, CASE_DISPLAY_NAME_ELEMENT_NAME, true);
+                caseDisplayName = getElementTextContent(caseElement, CASE_DISPLAY_NAME_ELEMENT_NAME, true);
+            }
+            caseNumber = getElementTextContent(caseElement, CASE_NUMBER_ELEMENT_NAME, false);
+            String examinerName = getElementTextContent(caseElement, EXAMINER_ELEMENT_NAME, false);
+            String examinerPhone;
+            String examinerEmail;
+            String caseNotes;
+            try {
+                examinerPhone = getElementTextContent(caseElement, EXAMINER_ELEMENT_PHONE, false);
+                examinerEmail = getElementTextContent(caseElement, EXAMINER_ELEMENT_EMAIL, false);
+                caseNotes = getElementTextContent(caseElement, CASE_ELEMENT_NOTES, false);
+            } catch (CaseMetadataException ex) {
+                examinerPhone = "";  //case had metadata file written before additional examiner details were included 
+                examinerEmail = "";
+                caseNotes = "";
```

Comment:
We should do a version number conditional here, as in the existing code, keeping this code consistent, and reserving exceptions for error conditions.